spec/system/candidate_interface/entering_details/candidate_entering_personal_details_when_requiring_visa_2022_without_entry_date_spec.rb

```diff
@@ -61,13 +61,13 @@ def and_i_can_complete_personal_information_stating_that_i_need_a_visa_sponsorsh
     expect(page).to have_content 'Name'
     expect(page).to have_content 'Lando Calrissian'
     expect(page).to have_content 'Pakistani'
-    expect(page).to have_content "Do you have the right to work or study in the UK?\nNot yet"
+    expect(page).to have_content "Do you have the right to work or study in the UK?\nNo"
```

Comment:
Think this can go back to how it was? (Although both will pass as "Not yet" starts with "No" 😂 )

```suggestion
    expect(page).to have_content "Do you have the right to work or study in the UK?\nNot yet"
```

Reply:
Great spot! Thanks, have updated